src/layouts/Static.tsx

```diff
@@ -111,13 +111,15 @@ export const StaticLayout = ({
             <Stack className="gap-8">
               <Breadcrumbs slug={slug} />
 
-              <p
-                className="text-body-medium"
-                dir={isLangRightToLeft(locale as Lang) ? "rtl" : "ltr"}
-              >
-                <Translation id="page-last-updated" />:{" "}
-                {lastEditLocaleTimestamp}
-              </p>
+              {slug !== "/whitepaper" && !asPath.includes("/whitepaper") && (
```

Comment:
```suggestion
              {!asPath.includes("/whitepaper") && (
```

The slug check is redundant, so going to remove that one